packages/core/scripts/identifier-umip/1_Propose.js

```diff
@@ -9,14 +9,26 @@ const { getTruffleContract } = require("../../index");
 const Governor = getTruffleContract("Governor", web3, "1.1.0");
 const IdentifierWhitelist = getTruffleContract("IdentifierWhitelist", web3, "1.1.0");
 
+const { GasEstimator } = require("@uma/financial-templates-lib");
+
 const argv = require("minimist")(process.argv.slice(), { string: ["identifier"] });
 
+const winston = require("winston");
+
 const proposerWallet = "0x2bAaA41d155ad8a4126184950B31F50A1513cE25";
 
 async function runExport() {
   console.log("Running Upgrade🔥");
   console.log("Connected to network id", await web3.eth.net.getId());
 
+  const gasEstimator = new GasEstimator(
+    winston.createLogger({
+      silent: true
```

Comment:
What does `silent` do? Why not just pass a `dummyLogger` which just prints to console:

```
dummyLogger = winston.createLogger({
            level: "info",
            transports: [new winston.transports.Console()]
          });
```

Reply:
It makes it print nothing. The bot-style logs that the gas estimator produces don't jive super well with the logs these scripts produce (they are super verbose and bulky). I can put the console logger in, but it just seems like it would distract from the other logs. Thoughts?